style.css

```diff
@@ -0,0 +1,152 @@
+* {
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  text-align: center;
+}
+
+.menu {
+  display: flex;
+  gap: 30px;
+  justify-content: center;
+  list-style: none;
+  margin: 20px;
+}
+
+.h-menu {
+  display: none;
+}
+
+.body h1 {
+  color: #172b4d;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+}
+
+.body h2 {
+  color: #36b37e;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  margin-top: 10px;
+}
+
```

Comment:
-[ ] Good job making most of the changes suggested by the previous reviewer 👏 . however, there's one more you need to work on. the background image in the middle, by the right of the header seems a little bit pused inside . Kindly fix it properly to resemble the specification as in the template given 👏 

## View:
![image](https://user-images.githubusercontent.com/50990925/218939177-029392eb-ddb3-4d08-a850-802f87bd7695.png)

## Template:
![image](https://user-images.githubusercontent.com/50990925/218939368-aa001168-3f25-4f47-96f8-a32958e763ea.png)